README.md

```diff
@@ -48,10 +48,10 @@ $ git clone https://github.com/apache/rocketmq-operator.git
 $ cd rocketmq-operator
 ```
 
-2. To deploy the RocketMQ Operator on your Kubernetes cluster, please run the following script:
+2. To deploy the RocketMQ Operator on your Kubernetes cluster, please run the following command:
 
 ```
-$ ./install-operator.sh
+$ make deploy
```

Comment:
Should we also wrap the command in `install-operator.sh`?

Reply:
OK, i will recover this file to keep usage compatibility.